src/model/OutputControlFiles.cpp

```diff
@@ -0,0 +1,628 @@
+}
+
+bool OutputControlFiles::setOutputMDD(bool outputMDD) {
+  return getImpl<detail::OutputControlFiles_Impl>()->setOutputMDD(outputMDD);
+}
+
+bool OutputControlFiles::setOutputMTD(bool outputMTD) {
+  return getImpl<detail::OutputControlFiles_Impl>()->setOutputMTD(outputMTD);
+}
+
+bool OutputControlFiles::setOutputEND(bool outputEND) {
+  return getImpl<detail::OutputControlFiles_Impl>()->setOutputEND(outputEND);
+}
+
+bool OutputControlFiles::setOutputSHD(bool outputSHD) {
+  return getImpl<detail::OutputControlFiles_Impl>()->setOutputSHD(outputSHD);
+}
+
+bool OutputControlFiles::setOutputDFS(bool outputDFS) {
+  return getImpl<detail::OutputControlFiles_Impl>()->setOutputDFS(outputDFS);
+}
+
+bool OutputControlFiles::setOutputGLHE(bool outputGLHE) {
+  return getImpl<detail::OutputControlFiles_Impl>()->setOutputGLHE(outputGLHE);
+}
+
+bool OutputControlFiles::setOutputDelightIn(bool outputDelightIn) {
+  return getImpl<detail::OutputControlFiles_Impl>()->setOutputDelightIn(outputDelightIn);
+}
+
+bool OutputControlFiles::setOutputDelightELdmp(bool outputDelightELdmp) {
+  return getImpl<detail::OutputControlFiles_Impl>()->setOutputDelightELdmp(outputDelightELdmp);
+}
+
+bool OutputControlFiles::setOutputDelightDFdmp(bool outputDelightDFdmp) {
+  return getImpl<detail::OutputControlFiles_Impl>()->setOutputDelightDFdmp(outputDelightDFdmp);
+}
+
+bool OutputControlFiles::setOutputEDD(bool outputEDD) {
+  return getImpl<detail::OutputControlFiles_Impl>()->setOutputEDD(outputEDD);
+}
+
+bool OutputControlFiles::setOutputDBG(bool outputDBG) {
+  return getImpl<detail::OutputControlFiles_Impl>()->setOutputDBG(outputDBG);
+}
+
+bool OutputControlFiles::setOutputPerfLog(bool outputPerfLog) {
+  return getImpl<detail::OutputControlFiles_Impl>()->setOutputPerfLog(outputPerfLog);
+}
+
+bool OutputControlFiles::setOutputSLN(bool outputSLN) {
+  return getImpl<detail::OutputControlFiles_Impl>()->setOutputSLN(outputSLN);
+}
+
+bool OutputControlFiles::setOutputSCI(bool outputSCI) {
+  return getImpl<detail::OutputControlFiles_Impl>()->setOutputSCI(outputSCI);
+}
+
+bool OutputControlFiles::setOutputWRL(bool outputWRL) {
+  return getImpl<detail::OutputControlFiles_Impl>()->setOutputWRL(outputWRL);
+}
+
+bool OutputControlFiles::setOutputScreen(bool outputScreen) {
+  return getImpl<detail::OutputControlFiles_Impl>()->setOutputScreen(outputScreen);
+}
+
+bool OutputControlFiles::setOutputExtShd(bool outputExtShd) {
+  return getImpl<detail::OutputControlFiles_Impl>()->setOutputExtShd(outputExtShd);
+}
+
+bool OutputControlFiles::setOutputTarcog(bool outputTarcog) {
+  return getImpl<detail::OutputControlFiles_Impl>()->setOutputTarcog(outputTarcog);
+}
+
+/// @cond
+OutputControlFiles::OutputControlFiles(std::shared_ptr<detail::OutputControlFiles_Impl> impl)
+  : ModelObject(std::move(impl))
+{}
+OutputControlFiles::OutputControlFiles(Model& model)
+  : ModelObject(OutputControlFiles::iddObjectType(),model)
+{
```

Comment:
Really a cosmetic thing, but I prefer when the Ctors are at the begining of the public declarations (right after the end of the detail (Impl) namespace block). That's where I go to look for them usually. I don't mind though.